google_takeout_parser/parse_html/activity.py

```diff
@@ -53,6 +53,8 @@ def _group_by_brs(els: TextOrEl) -> ListOfTags:
 
 def _parse_subtitles(
     subtitle_cell: bs4.element.Tag,
+    *,
+    file_dt: Optional[datetime],
```

Comment:
could default to `None` here, but they are not exposed to the public/only used in tests, so figured safer to be verbose

Reply:
yeah, sounds good